client/src/components/Common/DotsAnimation/__tests__/index.spec.js

```diff
@@ -4,7 +4,7 @@ import { mount } from 'enzyme';
 
 import DotsAnimation from '../../DotsAnimation';
 
-jest.mock('../../../Common/Canvas', () => 'Canvas');
```

Comment:
I decided to change dots by absolute import only if it has more than 2 levels depth (3 sets of dots and more).
For `../..` I think it does not make sense to change it for `components` as it is much longer than just 4 dots... 🤔